src/order/well_founded.lean

```diff
@@ -92,134 +69,226 @@ begin
     exact lt_of_le_of_ne xle (ne.symm ngt) }
 end
 
+-- TODO: rewrite this in terms of `well_founded_gt`.
 theorem well_founded_iff_has_max' [partial_order α] : (well_founded ((>) : α → α → Prop) ↔
```

Comment:
This seems like a very dumb restatement of the above theorem. Removing it would be a nontrivial breaking change, should I do it in another PR?